### PR TITLE
added documentation

### PR DESCRIPTION
**Pull request by neelSharma12**
## Summary
- Added documentation

### CODE REVIEW

File: AdyenTwint/AdyenTwint.docc/AdyenTwint.md [hunk 1]
```diff
@@ -1,13 +1,112 @@
 # ``AdyenTwint``
 
-<!--@START_MENU_TOKEN@-->Summary<!--@END_MENU_TOKEN@-->
+This module is used for supporting the Twint payment method through app via the `TwintComponent`. In order to support Twint payment via the Twint App, make sure to include this module in your application.
```
Comment (goergisn) [inline AdyenTwint/AdyenTwint.docc/AdyenTwint.md]:
```suggestion
This module is used for supporting the Twint payment method through the native Twint SDK via the `TwintComponent`. In order to support Twint payments via the native Flow, make sure to include this module in your application.
```

File: AdyenTwint/AdyenTwint.docc/AdyenTwint.md [hunk 2]
```diff
@@ -1,13 +1,112 @@
...
 
 ## Overview
 
-<!--@START_MENU_TOKEN@-->Text<!--@END_MENU_TOKEN@-->
+The flow for Twint via App starts with the `TwintComponent`. After pressing the button on the component, `TwintComponent` makes a `/payments` call with `subtype: sdk` and trigger the flow with retrieving the code through `ActionComponent`.
```
Comment (goergisn) [inline AdyenTwint/AdyenTwint.docc/AdyenTwint.md]:
```suggestion
The flow for Twint via the native SDK starts with the `TwintComponent`. After pressing the button on the component, `TwintComponent` makes a `/payments` call with `subtype: sdk` and trigger the flow with retrieving the code through `ActionComponent`.
```

File: AdyenTwint/AdyenTwint.docc/AdyenTwint.md [hunk 3]
```diff
@@ -1,13 +1,112 @@
...
+
+Your app must configure a `callbackAppScheme` so that it is reopened once the payment is done.
+
+```
+componentConfig.actionComponent.twint = .init(callbackAppScheme: yourReturnURLScheme)
+```
+
+### Note:
+
+1. Your app should implement the callback in the `AppDelegate.m` by adding below method.
```
Comment (goergisn) [inline AdyenTwint/AdyenTwint.docc/AdyenTwint.md]:
```suggestion
1. Your app should call the `RedirectComponent` in the `AppDelegate` by adding below method.
```

File: AdyenTwint/AdyenTwint.docc/AdyenTwint.md [hunk 4]
```diff
@@ -1,13 +1,112 @@
...
+
+```
+- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url
```
Comment (goergisn) [inline AdyenTwint/AdyenTwint.docc/AdyenTwint.md]:
```swift
func application(_ app: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey: Any] = [:]) -> Bool {
    RedirectComponent.applicationDidOpen(from: url)
    return true
}
```

**Review by goergisn — APPROVED** [review]
Looks good, just some tiny remarks but might go through a docs review anyway 👍

File: AdyenTwint/AdyenTwint.docc/AdyenTwint.md (hunk 1 above)
Comment (neelSharma12) [reply AdyenTwint/AdyenTwint.docc/AdyenTwint.md]: Good point

**Review by goergisn — APPROVED** [review]

**Pull request merged**